rancher/dashboard
# Fix sticky footer helm yaml tab

### Summary
Fixes #7975

Fixed sticky footer in Helm Chart Install Yaml Values editor

### How to test

Go to cluster tools, OPA Gatekeeper and install, click next to get the Values step.
The footer container the wizard buttons should be sticky as in the previous step:
